packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -551,12 +589,21 @@
         line-height: 1.5;
         text-align: left;
 
-        &-multiplier {
+        &-trade-param {
             font-size: 1rem;
             line-height: 1rem;
             color: var(--text-less-prominent);
         }
     }
+    // &__type-turbos {
```

Comment:
@kate-deriv please remove the commented-out code if we don't need it)

Reply:
Removed, thank you